packages/@react-aria/utils/src/getScrollParent.ts

```diff
@@ -11,14 +11,18 @@
  */
 
 export function getScrollParent(node: Element): Element {
+  if (isScrollable(node)) {
+    node = node.parentElement;
+  }
```

Comment:
Modified this so we don't just return the same node as its scroll parent if it happens to be scrollable. Caused a infinite loop [here](https://github.com/adobe/react-spectrum/blob/3039b89b78730e7f825ef5af36ed01e1fd8010e7/packages/%40react-aria/utils/src/scrollIntoView.ts#L98-L102) otherwise